src/ucs/arch/loongarch64/bitops.h

```diff
@@ -0,0 +1,34 @@
+/**
+* Copyright (C) Tactical Computing Labs, LLC. 2022. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_ARCH_LOONGARCH64_BITOPS_H_
```

Comment:
Can you please merge with identical rv64 bitops.h, maybe introducing bitops_generic.h ?